src/Tickets/Seating/Asset_Group_Path.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * The Seating Asset Group Path controller.
+ *
+ * @since TBD
+ *
+ * @package TEC\Controller;
+ */
+
+namespace TEC\Tickets\Seating;
+
+use TEC\Common\Contracts\Provider\Controller as Controller_Contract;
+use TEC\Common\StellarWP\Assets\Config;
+use Tribe__Tickets__Main as Tickets_Plugin;
+
+/**
+ * Class Asset_Group_Path.
+ *
+ * @since TBD
+ *
+ * @package TEC\Controller;
+ */
+class Asset_Group_Path extends Controller_Contract {
```

Comment:
Why a dedicated class? It's a lot of boilerplate for one line of logic. Could this be moved to the `Assets` controller?

Reply:
Yea makes sense. Added the single line i needed in the registration of the main seating controller.